src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/error/CourtDetailsParseException.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.error;
+
+
+public class CourtDetailsParseException extends RuntimeException {
+
+    private static final String message = "Failed to parse court details.";
+
+    public CourtDetailsParseException() {
```

Comment:
new exception class to improve error handling if someone breaks the court details JSON